json.go

```diff
@@ -0,0 +1,73 @@
+package clover
+
+import (
+	"os"
+	"encoding/json"
+	"bufio"
+	"io/ioutil"
+	"errors"
+)
+
+// Exports a `collection` to JSON format in `exportPath`
+func (db *DB) ExportCollection(collectionName string, exportPath string) error{
+	exists, err := db.HasCollection(collectionName)
+	if err != nil{
+		return err
+	}
+	if !exists{
+		return ErrCollectionNotExist
+	}
+
+	_, err = os.Stat(exportPath)
```

Comment:
I think lines 21-23 are not needed. If the path already exists, you will get an error from ioutil.WriteFile.

Reply:
The idea behind having this simple check on top is to avoid wasting unnecessary resources, for big collections this will help save time. What do you think?

Reply:
Yes, you are right. But it is also true that for now we are skipping more important optimizations. If we think about big collections, it is even more important to serialize object progressively, using the `ForEach()` method to iterate documents. Since this early implementation focuses on simplicity, I would remove the check